tests/test_smoke.py

```diff
@@ -832,6 +832,26 @@ def test_public_bucket(self, tmp_public_storage_obj, store_type):
         out = subprocess.check_output(['sky', 'storage', 'ls'])
         assert tmp_public_storage_obj.name not in out.decode('utf-8')
 
+    @pytest.mark.parametrize(
+        'nonexist_bucket',
+        [f's3://{str(uuid.uuid4())}', f'gs://{str(uuid.uuid4())}'])
+    def test_nonexistent_bucket(self, nonexist_bucket):
+        # Attempts to create fetch a stroage with a non-existent source.
+        with pytest.raises(
+                sky.exceptions.StorageBucketGetError,
+                match='Attempted to connect to a non-existent bucket'):
+            storage_obj = storage_lib.Storage(source=nonexist_bucket)
+
+    @pytest.mark.parametrize('private_bucket',
+                             [f's3://imagenet', f'gs://imagenet'])
+    def test_private_bucket(self, private_bucket):
+        # Attempts to access private buckets not belonging to the user.
+        # These buckets are known to be private, but may need to be updated if
+        # they are removed by their owners.
+        with pytest.raises(sky.exceptions.StorageBucketGetError,
+                           match='the bucket name is taken'):
```

Comment:
`match=backend_utils._BUCKET_FAIL_TO_CONNECT_MESSAGE.format(...)`

Reply:
Ahh private buckets trigger this message on gcp, so the common message is `the bucket name is taken`:
https://github.com/skypilot-org/skypilot/blob/619655258e7b1479e5144d5a364257ac5dba5e4d/sky/data/storage.py#L1129-L1133

Reply:
In this case, can we move this message as an variable such as `PRIVATE_BUCKET_ERROR`?